core/tests/protractor_desktop/playTranslations.js

```diff
@@ -0,0 +1,103 @@
+  var explorationPlayerPage = null;
+  var libraryPage = null;
+
+  beforeAll(async function() {
+    creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    explorationEditorTranslationTab =
+      explorationEditorPage.getTranslationTab();
+    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
+    explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    libraryPage = new LibraryPage.LibraryPage();
+
+    await users.createUser('testTranslations@translations.com',
+      'testTranslations');
+    await users.login('testTranslations@translations.com');
+    await workflow.createExploration();
+    await explorationEditorMainTab.exitTutorial();
+    await explorationEditorMainTab.setStateName('First');
+    await explorationEditorMainTab.setContent(await forms.toRichText(
+      'This is the first card.'));
+    await explorationEditorMainTab.setInteraction('EndExploration');
+    await explorationEditorPage.navigateToTranslationTab();
+    await explorationEditorTranslationTab.uploadAudioFileForLanguage(
+      'Hindi', '../data/cafe.mp3');
+    await explorationEditorPage.saveChanges();
+    await explorationEditorPage.navigateToSettingsTab();
+    await explorationEditorSettingsTab.setTitle('Test Translations');
+    await explorationEditorSettingsTab.setCategory('Languages');
+    await explorationEditorSettingsTab.setLanguage('English');
+    await explorationEditorSettingsTab.setObjective(
+      'Testing if translations works');
+    await explorationEditorPage.saveChanges('Done.');
+    await workflow.publishExploration();
+    await users.logout();
+  });
+
+  it('should play and pause audio translations', async function() {
+    await users.login('testTranslations@translations.com');
+    await libraryPage.get();
+    await libraryPage.playExploration('Test Translations');
+    await explorationPlayerPage.clickAudioBar();
+    await explorationPlayerPage.playAudio();
+    await explorationPlayerPage.expectAudioToBePlaying();
+    await explorationPlayerPage.pauseAudio();
+    await explorationPlayerPage.expectAudioToBePaused();
+  });
+
+  it('should play translations for multiple languages', async function() {
+    await browser.get('/');
+    await creatorDashboardPage.get();
+    await creatorDashboardPage.editExploration('Test Translations');
+    await explorationEditorPage.navigateToTranslationTab();
+    await explorationEditorTranslationTab.uploadAudioFileForLanguage(
+      'Arabic', '../data/ambient-noise.mp3');
+    await explorationEditorPage.saveChanges('Added another translation');
+    await libraryPage.get();
+    await libraryPage.playExploration('Test Translations');
+    await browser.refresh();
+    await explorationPlayerPage.clickAudioBar();
+    await explorationPlayerPage.changeLanguage('Arabic');
```

Comment:
Can we also check that the Hindi voiceover still works?

Reply:
done!